src/compas_rhino/install.py

```diff
@@ -109,6 +111,38 @@ def install(version=None, packages=None):
         sys.exit(exit_code)
 
 
+@compas.plugins.plugin(category='install', pluggable_name='installable_rhino_packages', tryfirst=True)
+def default_installable_rhino_packages():
+    # While this list could obviously be hard-coded, I think
+    # eating our own dogfood and using plugins to define this, just like
+    # any other extension/plugin would be is a better way to ensure consistent behavior.
+    return ['compas', 'compas_rhino']
+
+
+@compas.plugins.pluggable(category='install', selector='collect_all')
+def installable_rhino_packages():
+    """Provide a list of packages to make available inside Rhino.
+
+    Extensions providing Rhino or Grasshopper features
+    can implement this pluggable interface to automatically
+    have their packages made available inside Rhino when
+    COMPAS is installed into it.
+
+    Examples
+    --------
+    >>> import compas.plugins
+    >>> @compas.plugins.plugin(category='install')
+    ... def installable_rhino_packages():
+    ...    return ['compas_fab']
+
+    Returns
+    -------
+    :obj:`list` of :obj:`str`
+        List of package names to make available inside Rhino.
+    """
```

Comment:
so in some other package you would basically write a plugin for this function that adds itself to the list of installable packages?

Reply:
yep, eg. compas_fab: https://github.com/compas-dev/compas_fab/pull/200